src/shapes/image.class.js

```diff
@@ -164,12 +164,12 @@
      * @param {CanvasRenderingContext2D} ctx Context to render on
      */
     _stroke: function(ctx) {
-      ctx.save();
-      this._setStrokeStyles(ctx);
```

Comment:
_stroke is called by the _render function, _setStrokeStyle has been already done.